actionpack-action_caching.gemspec

```diff
@@ -2,21 +2,22 @@
 
 Gem::Specification.new do |gem|
   gem.name          = 'actionpack-action_caching'
-  gem.version       = '1.1.0'
+  gem.version       = '2.0.0'
   gem.author        = 'David Heinemeier Hansson'
   gem.email         = 'david@loudthinking.com'
   gem.description   = 'Action caching for Action Pack (removed from core in Rails 4.0)'
   gem.summary       = 'Action caching for Action Pack (removed from core in Rails 4.0)'
   gem.homepage      = 'https://github.com/rails/actionpack-action_caching'
 
+  gem.required_ruby_version = '>= 2.2.2'
   gem.files         = `git ls-files`.split($/)
   gem.executables   = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
   gem.license       = 'MIT'
 
-  gem.add_dependency 'actionpack', '>= 4.0.0', '< 5.0'
+  gem.add_dependency 'actionpack', '~> 5.x'
```

Comment:
Is there anything about this patch that precludes using this gem with Rails 4?

If not, I’d suggest changing this constraint to:

```ruby
gem.add_dependency 'actionpack', '>= 4', '< 5.1'
```

or 

```ruby
gem.add_dependency 'actionpack', '>= 4', '< 6'
```